CMakeLists.txt

```diff
@@ -849,6 +849,26 @@ if (ZLIB_ENABLE_TESTS)
         add_test(example_dict_fuzzer example_dict_fuzzer${CMAKE_EXECUTABLE_SUFFIX} ${ALL_SRC_FILES})
         add_test(minigzip_fuzzer minigzip_fuzzer${CMAKE_EXECUTABLE_SUFFIX} ${ALL_SRC_FILES})
     endif()
+
+    # Use python script to translate exit code 0 or 1 as success
+    set(CVES CVE-2002-0059 CVE-2004-0797 CVE-2005-1849 CVE-2005-2096)
+    find_program(PYTHON_EXECUTABLE python)
```

Comment:
Wouldn't it be better to use python3?

Reply:
@iii-i We should not depend on specific version of python... It should work on any version...

Reply:
Python3 should be supported, but should not be a requirement.
Also, most distros symlink `python` to `python2` or `python3` depending on their choice of primary python version.

Reply:
@Dead2 My point exactly... it should support both python2 and python3, but through the symlink... Not having a symlink is not valid excuse to default to version that might not exist in all systems we are intending to support.